src/main/java/appeng/mixins/AnnihilationPlaneEnchantmentMixin.java

```diff
@@ -0,0 +1,20 @@
+package appeng.mixins;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.world.item.Item;
+
+import appeng.core.definitions.AEParts;
+
+@Mixin(targets = "net/minecraft/world/item/enchantment/EnchantmentCategory$7")
```

Comment:
This needs a class-javadoc that explains what the mixin does (why is it needed, basically).
This is important for porting to newer MC versions because no one will remember why this exists :-D

Reply:
Sorted.